src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
The `@Inject` annotation should also have a `@Target` annotation to specify where this annotation can be used. For example, you might want to restrict it to fields and constructor parameters.